example/src/App.tsx

```diff
@@ -44,15 +40,14 @@ export default function App() {
   const peripheral = new Peripheral()
 
   React.useEffect(() => {
-    const onDiscoverPeripheralListener = central.registerOnScannedListener(
+    const onDiscoverPeripheralListener = central.registerOnDiscoveredListener(
       ({ peripheralId: pId }: { peripheralId: string }) => {
```

Comment:
I don't think we have to supply the types here. Also, we don't have to abbreviate peripheralId.